persistence/migrations/1_initial.up.sql

```diff
@@ -29,7 +29,9 @@ CREATE TABLE invoices
 
         "final_cltv_delta"      INTEGER         NOT NULL CHECK (final_cltv_delta > 0),
         "payment_addr"          BYTEA           NOT NULL UNIQUE CHECK (LENGTH(payment_addr) = 32),
-        "payment_request"       TEXT            NOT NULL CHECK (LENGTH(payment_request) > 0)
+        "payment_request"       TEXT            NOT NULL CHECK (LENGTH(payment_request) > 0),
```

Comment:
looks like `payment_request` should be part of a different pr

Reply:
Why? This is just adding a comma, it was already in master?

Reply:
yes, my bad